package.json

```diff
@@ -43,22 +43,22 @@
 		"yauzl": "^2.10.0"
 	},
 	"devDependencies": {
-		"@commitlint/cli": "^12.1.4",
-		"@commitlint/config-conventional": "^12.1.4",
+		"@commitlint/cli": "^17.1.2",
+		"@commitlint/config-conventional": "^17.1.0",
 		"@seadub/danger-plugin-dependencies": "1.0.0",
 		"@seadub/danger-plugin-eslint": "^2.0.0",
 		"@seadub/danger-plugin-junit": "0.3.0",
-		"danger": "^10.9.0",
+		"danger": "^11.1.4",
 		"eslint": "^7.32.0",
 		"eslint-config-axway": "^6.0.2",
 		"eslint-plugin-mocha": "^9.0.0",
-		"fs-extra": "^9.1.0",
-		"husky": "^6.0.0",
-		"lint-staged": "^11.2.6",
+		"fs-extra": "^10.1.0",
```

Comment:
Not caused by this PR, but I _think_ we might want to move fs-extra and semver to `dependencies` as it appears they're incorrectly in `devDependencies` (looks like I put them there incorrectly a few years back... oops 😇 )

Reply:
moved them!